cmd/create/oidcconfig/cmd.go

```diff
@@ -516,7 +516,8 @@ func (s *CreateUnmanagedOidcConfigManualStrategy) execute(r *rosa.Runtime) {
 	if r.Reporter.IsTerminal() {
 		r.Reporter.Infof("Please run commands above to generate OIDC compliant configuration in your AWS account. " +
 			"After running the commands please refer to the documentation to register your unmanaged OIDC Configuration " +
-			"with OCM.")
```

Comment:
Should I remove "After running the commands please refer to the documentation" ? @gdbranco

Reply:
Made a happy medium change, let me know what you think

Reply:
I think it is okay to do that, but we actually should adjust so that these two matches
https://github.com/openshift/rosa/blob/08377c478b454db2c0037cbee25b3863d72ba491/cmd/create/oidcconfig/cmd.go#L393


Which means it is probably better to have a const unifying them

Reply:
Though, isn't L517 about just running commands? It seems to run at the end, so do we even need "try again through" for L517? :thinking: